homeassistant/components/insteon/api/__init__.py

```diff
@@ -42,3 +55,24 @@ def async_load_api(hass):
     websocket_api.async_register_command(hass, websocket_write_properties)
     websocket_api.async_register_command(hass, websocket_load_properties)
     websocket_api.async_register_command(hass, websocket_reset_properties)
+
+
+async def async_register_insteon_frontend(hass: HomeAssistant):
+    """Register the Insteon frontend configuration panel."""
+    dev_url = hass.data.get(DOMAIN, {}).get(CONF_DEV_URL)
+    if dev_url:
+        hass.http.register_view(InsteonFrontendDev(hass, dev_url))
+    else:
+        hass.http.register_static_path(URL_BASE, locate_dir(), cache_headers=False)
+    # Add to sidepanel if needed
+    if DOMAIN not in hass.data.get("frontend_panels", {}):
```

Comment:
The whole `async_register_insteon_frontend` function should only be called once, not just this panel registration.